### ISSUE TITLE
Alterac Valley scripting

### ISSUE THREAD

**Issue by Dualityz** [LightsHope/issues#1117]
2 times out of 3 the Ice Lord and Ivus will run through terrain back to the middle field after killing a unit, and then not move from there at all. Also, I'm not sure if it's intended but I ( along with a lot of people I'm sure ) have noticed instances of Horde rushing straight to Vanndar without killing any tower boss / Balinda, which should theoretically be impossible with the warmasters standing around; I have not seen Alliance be able to do the same ( skip Galv ) and succeed, ever. It's unclear if something is bugged  but it does feel very odd.

**Comment by Dualityz** [comment]
Adding to this, areas around Galv and Balinda will still get roaming NPCS defending it even when the boss is dead. This gets very problematic in the areas around Galv.
Defending NPC packs will keep spawning on towers and graveyards that are taken by the opposite faction. 
Both of these makes for some very awkward situations where its sometimes hard to defend or take a point even when outnumbering the enemy players by a lot. Becomes especially obvious if the game has lasted for long enough.
This is known as a "hole" to a variety of AV farmers : the more the game goes on, the more the NPCS ( including the ones that shouldnt be there ) are buffed up, making the game gets more and more stuck.

Still unsure if any of this existed in the vanilla AV.

**Comment by dinsfireyo** [comment]
i agree that you are unsure about how alterac valley works do sum research plox

**Comment by Dualityz** [comment]
Thanks for the input ! I absolutely am not sure about any of this, but some of it is very obviously bugs, IE guards spawning on opposite side capped GY and Ivus / Icelord behavior.

**Comment by dinsfireyo** [comment]
the guards will keep spawning until you have completely captured the graveyard, ivuz and forest lord will sit in mid for 10 minutes before they decide to push(not sure if they work, havent seen it yet)
